helpers/collection/collection.go

```diff
@@ -8,83 +8,101 @@ import (
 	"github.com/go-pg/pg/orm"
 )
 
-const defaultOrdersLimit = 25
+// DefaultItemsLimit per page limit
+var DefaultItemsLimit = 50
+
+// SeparatorOrder
+var SeparatorOrder = ","
+
+type (
+	// Former interface
+	Former interface {
+		ApplyFilter(*Options) error
+		ApplyPager(*Options) error
+		ApplySorter(*Options) error
+	}
 
-// Filter interface
-type Filter interface {
-	Pager() Paginate
-	Apply(*orm.Query) error
-}
+	// Fetcher closure to fetch and format Items
+	Fetcher = func(*Options) (Items, error)
 
-// Paginate is part of filter structure
-type Paginate struct {
-	Offset int `query:"offset" form:"offset" json:"offset"`
-	Limit  int `query:"limit" form:"limit" json:"limit"`
-}
+	// Items is formatted response of models
+	Items = []interface{}
 
-// Pager returns Paginate-struct
-func (p Paginate) Pager() Paginate {
-	var limit = defaultOrdersLimit
+	// Collection response answer
+	Collection struct {
+		Total  int   `json:"total"`
+		Offset int   `json:"offset"`
+		Items  Items `json:"items"`
+	}
 
-	if p.Limit > 0 {
-		limit = p.Limit
+	// Options to Format list-response
+	Options struct {
+		Query   *orm.Query
+		Fetcher Fetcher
+		Former  Former
 	}
+)
 
-	return Paginate{
-		Offset: p.Offset,
-		Limit:  limit,
+// ResponseQuery collection as response-answer (Collection)
+func ResponseQuery(ctx web.Context, opts Options) error {
```

Comment:
I'm vote for Response, without Query
since it makes no sense in Query part of naming